tests/test_client.py

```diff
@@ -34,6 +34,10 @@
     TradeState,
 )
 
+# Set the year of the delivery starts to one year from now to make sure
+# we never get an expired delivery start
+NEXT_YEAR = datetime.now().year + 1
```

Comment:
you might want to add timezone information

Reply:
I actually ended up removing the constant as it was only used once, and set the delivery start to the next day and not next year as it is more realistic.